README.md

```diff
@@ -259,7 +260,12 @@ prefix `OCaml:`:
 | `ocaml.current-dune-file`    | Open Dune File (located in the same folder) |                    |
 | `ocaml.switch-impl-intf`     | Switch implementation/interface             | `Alt+O`            |
 | `ocaml.open-repl`            | Open REPL                                   |                    |
-| `ocaml.evaluate-selection`   | Evaluate Selection                          | `Shift+Enter`      |
+| `ocaml.evaluate-expression`  | Evaluate Expression in REPL                 | `Shift+Enter`      |
+| `ocaml.evaluate-file`        | Evaluate Selection                          | `Shift+Alt+Enter`  |
+
+Note: `ocaml.evaluate-expression` was previously named
+`ocaml.evaluate-selection`. You can also disabe the new cursor jumps after this
```

Comment:
Why don't we preserve `ocaml.evaluate-selection`? I don't see anything wrong with having both commands.

Reply:
I understood. I just don't see why we should remove a command with a clear name and semantics and replace it with a command that has neither of those things. Seems like we're just irritating users who are used to the old command this way.

Reply:
Good point, but then I would show a deprecation message saying to use the new command because 

1. The new command has almost the same semantics - select and evaluate (no selection resulted in evaluating the whole line but I'm not sure that's very useful and the new command does the same thing if it's a single-line let binding) 
2. There's then contention for the shortcut: keep the old and miss out on a better command or replace and still irritate